app/models/datastreams/page_desc_metadata.rb

```diff
@@ -0,0 +1,17 @@
+class PageDescMetadata < ActiveFedora::NtriplesRDFDatastream
+
+  include BasicRdfProperties
+
+  # Define custom properties not in basic vocabulary definitions
+  map_predicates do |map|
+    map.paged_struct(in: RDF::DC, to: "isPartOf") do |index|
```

Comment:
I think the attribute being mapped at this line has to be page_struct to match line 27 in page.rb.